modules/regulus/pages/getting-started-with-regulus.adoc

```diff
@@ -2,11 +2,11 @@
 :experimental:
 :page-author: Thripti Aravind
 :page-email: thripti.aravind@teradata.com
-:page-revdate: April 10th, 2023
+:page-revdate: May 16th, 2023
 :description: Run a simple workflow and learn how to use Regulus.
 :keywords: Regulus, Workspaces, Query Engine
 
-IMPORTANT: This product is in preview and is subject to change. To get early access to Regulus, contact Support.
+IMPORTANT: This product is in preview and is subject to change. To get early access to Regulus, sign up on the Regulus Home page.
```

Comment:
@thripti , the url for the Regulus page is already known: https://developers.teradata.com/regulus/. We have an empty placeholder page right now. I'll let you guys figure out when you want to link to https://developers.teradata.com/regulus/.